src/utils/redirect.ts

```diff
@@ -6,9 +6,15 @@ import { useActiveWeb3React } from 'hooks'
 import { useChangeNetwork } from 'hooks/web3/useChangeNetwork'
 
 const whiteListDomains = [/https:\/\/(.+?\.)?kyberswap\.com$/, /https:\/\/(.+)\.kyberengineering\.io$/]
-export const validateRedirectURL = (url: string | undefined, whitelistKyberSwap = true) => {
+
+type Options = { whitelistKyberSwap?: boolean; allowPath?: boolean }
+export const validateRedirectURL = (
+  url: string | undefined,
+  { whitelistKyberSwap = true, allowPath = false }: Options = {},
+) => {
   try {
     if (!url) throw new Error()
+    if (allowPath && url.startsWith('/')) return url
```

Comment:
How about `./` links?

Reply:
no use case in our src code, don't support for now.